src/etsn20_lab4/str_search.java

```diff
@@ -6,9 +6,9 @@
 	public static void main(String[] args) {
 		
 	    Scanner userInput = new Scanner(System.in);
```

Comment:
No need to use Scanners. You can use the String[] args. Or atleast give the user the option to use them